ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -363,13 +383,8 @@ class ServicesHomeScreen extends React.Component<Props, State> {
   }
 
   private onServiceSelect = (service: ServicePublic) => {
-    // when a service gets selected, before navigating to the service detail
-    // screen, we issue a contentServiceLoad to refresh the service metadata
-
-    // TODO: evaluate if it makes sense to load the service metadata also on click  (now they are previously loaded)
-    this.props.contentServiceLoad(service.service_id);
```

Comment:
we could remove **contentServiceLoad**  from mapDispatchToProps since it is no longer nedeed

Reply:
updated within https://github.com/teamdigitale/io-app/pull/1352/commits/210c4c3c9c27f94e872033365c0dda7db78b14b1